tests/test_extraction/test_embedded_tweet_extractor.py

```diff
@@ -0,0 +1,99 @@
+"""
+    This script is for unit testing of embedded_tweet_extractor
+    Use pytest to run this script
+    Command to run: /stampify$ python -m pytest
+"""
+import bs4
+import pytest
+
+from extraction import embedded_tweet_extractor
+from extraction.data_models.embedded_tweet import ETweet
+
+__EXTRACTOR = embedded_tweet_extractor.ETweetExtractor()
+
+
+@pytest.fixture(scope='module')
+def soup():
```

Comment:
**TODO:** Move this method to test_utils.

Reply:
Can you do move it?

Reply:
I will do it once all modules are merged.